compute_tools/src/http/openapi_spec.yaml

```
@@ -395,6 +409,24 @@ components:
         - configuration
       example: running
 
+    InstalledExtensions:
+      type: object
+      properties:
+        extensions:
+          description: Contains list of installed extensions.
+          type: array
+          items:
+            type: object
+            properties:
+              name:
```

```suggestion
              extname:
```
to match the InstalledExtensions struct.

Is there some tool to verify the openapi spec against the rust structs?